drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -272,5 +321,4 @@ int lis2dh_acc_slope_config(const struct device *dev,
 int lis2dh_spi_init(const struct device *dev);
 int lis2dh_i2c_init(const struct device *dev);
 
-
-#endif /* __SENSOR_LIS2DH__ */
+#endif /* __SENSOR_LIS2DH__ */
```

Comment:
check if the blank lines are added to the files where required

Reply:
How would i check this? I put the blank line back for now.

Reply:
if you go to the 'Files Changed' tab in the PR, you'll see some red symbols (circle with a stripe) at the end of some files, indicating the lack of a blank line